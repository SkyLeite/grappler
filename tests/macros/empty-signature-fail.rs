#[grappler::hook(signature = "")]
pub fn foo() {}

pub fn main() {}
