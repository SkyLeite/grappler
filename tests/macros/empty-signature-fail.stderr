error: custom attribute panicked
 --> tests/macros/empty-signature-fail.rs:1:1
  |
1 | #[grappler::hook(signature = "")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Signature cannot be empty
